### PR TITLE
Arentas/blazor init

### PR DESCRIPTION
**Pull request by ArentasMeinorius**
Initial commit of the app

### CODE REVIEW

**Comment by ArentasMeinorius** [comment]
@cezaris13 maybe you could make at least 1 approve on PR required?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Team members:
+arentas, martinas, pijus
```
Comment (cezaris13) [inline README.md:2]: Capitalize names

**Comment by cezaris13** [comment]
> @cezaris13 maybe you could make at least 1 approve on PR required?

@ArentasMeinorius added branch protection rule

**Comment by cezaris13** [comment]
> > @cezaris13 maybe you could make at least 1 approve on PR required?
> 
> @ArentasMeinorius added branch protection rule

**Review by cezaris13 — APPROVED** [review]

**Pull request merged**